extension/content.js

```diff
@@ -409,6 +409,13 @@ function fixSquashAndMergeTitle() {
 	});
 }
 
+function addTitleToEmojis() {
+	const emojis = $('.js-comment-body, #readme').find('g-emoji');
```

Comment:
This would not cover commits (message and body), markdown documents other than the readme and others. Why not just `$('g-emoji')`?